.github/workflows/deploy-full-website.yml

```diff
@@ -0,0 +1,60 @@
+name: Manually Rebuild All Docs and Create a PR with Docs Changes
+on:
+    workflow_dispatch:
+jobs:
+  build_docs:
+    name: Build and Deploy Documentation
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0 # to ensure other branches are available for build
+    - name: Install system dependencies
+      run: sudo apt-get install -y libhdf5-dev
+    - name: Set r compilation options
+      run: bash -c 'echo -e "MAKEFLAGS=--jobs=3\nCXXFLAGS += -O1 -UNDEBUG" > "$GITHUB_WORKSPACE/Makevars.user" && echo "R_MAKEVARS_USER=$GITHUB_WORKSPACE/Makevars.user" >> "$GITHUB_ENV"'
+    - name: Setup R
+      uses: r-lib/actions/setup-r@v2
+    - name: Install R dependencies
+      uses: r-lib/actions/setup-r-dependencies@v2
+      with: 
+        cache-version: 1
+        working-directory: 'r'
+        extra-packages: |
+            any::pkgdown
+            any::devtools
+            any::uwot
+            any::irlba
+            any::RcppHNSW
+            any::igraph
+            any::BiocManager
+            bioc::BSgenome.Hsapiens.UCSC.hg38
+            github::GreenleafLab/motifmatchr
+            github::GreenleafLab/chromVARmotifs
```

Comment:
It looks like we can actually get these packages out of the workflow and into our `DESCRIPTION` file under a section `Config/Needs/website:`. See [r-pkgs book](https://r-pkgs.org/dependencies-in-practice.html#sec-dependencies-nonstandard-config-needs) and [ggplot2 github action](https://github.com/tidyverse/ggplot2/blob/f12b73de2609603ff405360c73f1a5f38466af45/.github/workflows/pkgdown.yaml#L35-L38). From [searching github](https://github.com/search?q=%22Config%2FNeeds%2Fwebsite%3A%22+path%3ADESCRIPTION+&type=code), it appears that it's possible to list these one-per-line without commas, e.g. as [tidymodels/workflows](https://github.com/tidymodels/workflows/blob/main/DESCRIPTION) does.

Reply:
Apparently there's some added complexity for importing from non-CRAN sources.  I added everything else in that didn't have a `bioc` or `github` label into `DESCRIPTION` as suggested!